### ISSUE TITLE
Exported file contains CRLF as line endings

### ISSUE THREAD

**Issue by mbrembilla** [qwazer/scheme2ddl#54]
Hi all,
I run scheme2ddl on a machine with Unix and I noticed that the exported files contains the Windows line endings (CRLF).
Can please someone tell if I'm doing something wrong or if there is a option to avoid this behaviour?

I would prefer not to refactor all the files after the export, if it's possible.

Have a nice day
--
Mauro

**Comment by qwazer** [comment]
It's not configurable now. caused by https://github.com/qwazer/scheme2ddl/commit/24d0a401d0f8fc0ac4b8ef9e1343e55f3c8219bb

**Comment by arno82** [comment]
yeah, I invented this behaviour, but you are right, it should be configurable. As I don't use scheme2dll anymore I think it's up to qwazer to change it.
